test/test-gotest.sh

```diff
@@ -25,6 +31,10 @@ function run-test()
 	$XSUDO $@ || failures=$( [ -n "$failures" ] && echo "$failures\\n$@" || echo "$@" )
 }
 
+if [[ "$TAGS" != '' ]];then
+	XTAGS="-tags $TAGS"
```

Comment:
I think you might want the equivalent of:
```
-tags "tag1 tag2"
```

for this to work correctly with more than one tag.

Reply:
Indeed, stumbled upon another bug as well on a second review. Should both be fixed now.